src/components/NoteDialog/NoteDialog.tsx

```diff
@@ -48,6 +49,12 @@ const NoteDialog = ({noteId, show, text, authorId, isAdmin, authorName, showAuth
     }
   };
 
+  const onUnstack = (id: string, authorId: string) => {
+    if (editable(authorId)) {
```

Comment:
I think every participant should be able to unstack notes from stacks

Reply:
Would look like this (normal user who did not create the note). I think this is a good idea.
<details open>
<summary>Unstack notes</summary>

![unstack](https://user-images.githubusercontent.com/60005702/132484215-5aa8e089-01f3-4403-a331-ce1ee9e346e3.png)

</details>

Reply:
👍🏼